UPGRADE-3.x.md

```diff
@@ -41,13 +41,18 @@ __construct($id, ThumbnailInterface $thumbnail, BackendInterface $backend, Event
 When creating a custom video provider, you have to implement the ``getReferenceUrl`` method to establish
 the media url.
 
+### Deprecations
```

Comment:
This placement is totally wrong and already reporter.

This must be on under the UPGRADE 3.x, not under an already released one.

We must take care of this to avoid weird git hack and ugly history.